app/views/cars/show.html.erb

```diff
@@ -1,44 +1,115 @@
+              <% end %>
+            <% else %>
+              <% if @car.sell_price %>
+                <div class="d-flex align-items-center justify-content-between">
+                  <p class = "new-price">Sell Price: <span><%=@car.sell_price%></span></p>
+                </div>
+              <% end %>
+              <% if @car.rent_price %>
+                <div class="d-flex align-items-center justify-content-between mt-1">
+                  <p class = "last-price">Rent Price: <span><%= @car.rent_price%></span></p>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+          <div class = "product-detail">
+            <h2>Description </h2>
+            <p><%= @car.car_description %></p>
+            <ul>
+              <li>Model: <span><%= @car.purchase_date.year%></span></li>
+              <li>Car Type: <span><%= @car.car_type%></span></li>
+              <li>Transmission Type: <span><%= @car.transmission_type%></span></li>
+              <li>Engine: <span><%= @car.engine_type%></span></li>
+              <li>Capacity: <span><%= @car.seats%> seater</span></li>
+              <li>DIstance Driven: <span><%= @car.distance_driven%> KMs</span></li>
+            </ul>
+          </div>
+          <div class = "social-links">
+            <p>Share At: </p>
+            <a href = "#">
+              <i class = "fab fa-facebook-f"></i>
+            </a>
+            <a href = "#">
+              <i class = "fab fa-twitter"></i>
+            </a>
+            <a href = "#">
+              <i class = "fab fa-instagram"></i>
+            </a>
+            <a href = "#">
+              <i class = "fab fa-whatsapp"></i>
+            </a>
+            <a href = "#">
+              <i class = "fab fa-pinterest"></i>
+            </a>
+          </div>
+          <div>
+            <br>
+            <div>
+              <% if can? :update, @car%>
+                <%= link_to "Edit this car", edit_car_path(@car),class:"btn btn-outline-dark btn-sm me-2" %>
+              <% end %>
+              <%= link_to "Back to cars", cars_path,class:"btn btn-outline-primary btn-sm me-2" %>
+              <% if can? :destroy, @car%>
+                <%= link_to "Destroy this car", @car, method: :delete,class:"btn btn-outline-danger btn-sm" %>
+              <% end %>
+            </div>
+          </div>
+        </div>
       </div>
-    <% end %>
-    <p>
-    </p>
-  <% end %>
-</div>
-
-<!-- To add car_image -->
-<div class="border">
-<%= form_with(model: @car, local: true) do |form| %>
-  <p>
-    <%= form.label :car_images, "Upload New Photos:" %>
-    <%= form.file_field :car_images, multiple: true %>
-  </p>
-  <div class="actions">
-    <%= form.submit "Upload" , data: { turbo: false}%>
-  </div>
-<% end %>
-
-</div>
-<div>
-  <%= link_to "Edit this car", edit_car_path(@car) %> |
-  <%= link_to "Back to cars", cars_path %>
-  <%= button_to "Destroy this car", @car, method: :delete %>
-</div>
+    </body>
+</html>
```

Comment:
check indentation in lines 114 and 115. seems there's an extra tab. 
check for whole file